on-chain/uniswap_v4_stateful_trading_agent.py

```
@@ -39,7 +39,10 @@
     DEFAULT_ETH_ALLOCATION,
     USE_MLX_MODEL,
     MLX_BASE_MODEL,
-    MLX_ADAPTER_PATH
+    MLX_ADAPTER_PATH,
+    AVAILABLE_MODELS,
+    OPENROUTER_API_KEY,
+    USE_FORK
```

Ruff (0.11.9)

`config.USE_FORK` imported but unused

Remove unused import: `config.USE_FORK`

(F401)